packages/ETLoad.package/ETLOopMap.class/class/toSmallInteger..st

```diff
@@ -0,0 +1,5 @@
+interpretation
+toSmallInteger: anOop
+	"Converts the oop to the small integer it is referencing."
+	
+	^ anOop >> 1.
```

Comment:
* Is this guaranteed to produce a `SmallInteger`? 
* Does this handle negative numbers correctly?